app/static/src/scss/includes/_record.scss

```diff
@@ -1,5 +1,16 @@
 /* SCSS for Record page */
 
+.govuk-tabs__list-item {
```

Comment:
I have skipped over this file as well as other css files, so worth getting someone else to review.

Reply:
Will request another review from Lucian.